native/src/main/java/io/ballerina/stdlib/java.jms/LoggingExceptionListener.java

```diff
@@ -28,10 +27,10 @@
  * Logging exception listener class for JMS {@link javax.jms.Connection}.
  */
 public class LoggingExceptionListener implements ExceptionListener {
-    private static final Logger LOGGER = LoggerFactory.getLogger(LoggingExceptionListener.class);
+    private static final PrintStream ERR_OUT = System.err;
 
     @Override
     public void onException(JMSException connectionException) {
-        LOGGER.error("Connection exception received.", connectionException);
+        ERR_OUT.println("Connection exception received from the JMS provider: " + connectionException.getMessage());
```

Comment:
Shouldn't we print the stacktrace here?

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/140/commits/4389f20a5c835ed5d5bf36974b1e8edc43965a78